app/editor/src/features/content/list-view/constants/columns.tsx

```diff
@@ -20,43 +19,43 @@ export const columns: (Column<IContentModel> & UseSortByColumnOptions<IContentMo
   {
     id: 'mediaType',
     Header: 'Type',
-    width: 125,
+    maxWidth: 100,
```

Comment:
With the hard-coded values it stretches the page massively and doesn't support the combined view very well.  However, with this change it also doesn't correctly expand the headline column when it has space too.